app/templates/power-ups/topics.html

```diff
@@ -1,9 +1,54 @@
 <div class="app-content">
-  <h2>Menus</h2>
-  <p>Often you will want to render Menus to the member, such as in response to clicking on a card-button. This code is invoked
-    as the callback when a member clicks on the card-button returned as part of the plugin initialization. Initially the callback
-    runs the t.popup method and provides two hardcoded options. If the user clicks on one of these options, we then create
-    an overlay via t.overlay that loads the appropriate interface in a new iframe.</p>
+  <h1>Power-Up Topics</h1>
+  <ul>
+    <li><a href="#cors">CORS Settings</a></li>
+    <li><a href="#menus">Menus</a></li>
+    <li><a href="#attachments">Attachments</a></li>
+    <li><a href="#authentication">Authentication</a></li>
+    <li><a href="#data-storage">Data Storage</a></li>
+    <li><a href="#recoloring">Recoloring</a></li>
+    <li><a href="#timeouts">Timeouts</a></li>
+    <li><a href="#internationalization">Internationalization</a></li>
+    <li><a href="#callbacks">Callbacks</a></li>
+    <li><a href="#webhooks-and-offline-access">Webhooks &amp; Offline Access</a></li>
+    <li><a href="#troubleshooting">Power-Ups Troubleshooting</a></li>
+  </ul>
+
+  <h2 id="cors">CORS Settings</h2>
+  <p>It looks like your Power-Up can't be loaded because the CORS settings on your server. Because the request for this file is coming from trello.com, your server will need to allow for cross-origin requests from our domain.</p>
+
+  <p>Hosting it on Github pages works because they allow cross-site requests: http://stackoverflow.com/questions/26416727/cross-origin-resource-sharing-on-github-pages</p>
+
+  <p>If you're not sure how to change CORS settings, I'm happy to help you figure it out - just let me know what type of server you're running. The gomix example shows how to set CORS in the server.js file: https://gomix.com/#!/project/rune-friend</p>
```

Comment:
The gomix project link is outdated. Should now be: https://glitch.com/edit/#!/trello-power-up